tests/ast/data/RelationalGroupedDataFrame.test

```diff
@@ -915,6 +915,10 @@ body {
   assign {
     expr {
       sp_dataframe_sort {
+        ascending {
```

Comment:
Instead of `null_vals`, these can be omitted entirely if `ascending` is `Option[Expr]` instead of `Expr`. That requires changing the server-side code which I can do in a different PR.